Templates/Per Tenant Extension/.github/workflows/_BuildALGoProject.yaml

```diff
@@ -67,7 +67,7 @@ on:
         required: false
         type: string
 jobs:
-  BuildALGoProject:
+  I:
```

Comment:
Can't we set `name` instead?

Also, new idea: what is the nested workflow has the same name as the parent workflow?